test/python/expected/pyodbc/ddl_partitions.out

```diff
@@ -0,0 +1,510 @@
+GO
+
+CREATE PARTITION FUNCTION [DatePartitionFunction](date) AS RANGE RIGHT FOR VALUES (N'2022-01-01T00:00:00.000', N'2022-07-01T00:00:00.000', N'2023-01-01T00:00:00.000')
+GO
+
+CREATE PARTITION FUNCTION [DateTime2PartitionFunction](datetime2(6)) AS RANGE RIGHT FOR VALUES (N'2019-01-01T00:00:00.000', N'2022-01-01T00:00:00.000', N'2022-07-01T00:00:00.000')
+GO
+
+CREATE PARTITION FUNCTION [DateTimePartitionFunction](datetime) AS RANGE RIGHT FOR VALUES (N'2019-01-01T00:00:00.000', N'2022-01-01T00:00:00.000', N'2022-07-01T00:00:00.000')
+GO
+
+CREATE PARTITION FUNCTION [DecimalPartitionFunction](decimal(38,38)) AS RANGE RIGHT FOR VALUES (0.00000, 100.00000, 200.00000, 300.00000)
+GO
+
+CREATE PARTITION FUNCTION [IntPartitionFunction](int) AS RANGE RIGHT FOR VALUES (0, 500, 1000)
+GO
+
+CREATE PARTITION FUNCTION [MoneyPartitionFunction](money) AS RANGE RIGHT FOR VALUES (2.5679, 3.5679, 4.9100)
+GO
+
+CREATE PARTITION FUNCTION [NCharPartitionFunction](nchar(4000)) AS RANGE RIGHT FOR VALUES (N'A    ', N'B    ', N'F    ', N'K    ', N'P    ', N'U    ', N'Z    ')
+GO
+
+CREATE PARTITION FUNCTION [NumericPartitionFunction](numeric(38,38)) AS RANGE RIGHT FOR VALUES (0.00, 10.00, 20.00, 30.00)
+GO
+
+CREATE PARTITION FUNCTION [NVarCharPartitionFunction](nvarchar(4000)) AS RANGE RIGHT FOR VALUES (N'Apple', N'Banana', N'Cherry', N'Date', N'Mango')
+GO
+
+CREATE PARTITION FUNCTION [PARTITION_FUNCTION  유니코드스키마👻  ](int) AS RANGE RIGHT FOR VALUES (0, 500, 1000)
+GO
+
+CREATE PARTITION FUNCTION [PARTITION_FUNCTION 😎$@ #123 🌍rder  ](int) AS RANGE RIGHT FOR VALUES (0, 500, 1000)
+GO
+
+CREATE PARTITION FUNCTION [PARTITION_FUNCTIONログインαιώνια](int) AS RANGE RIGHT FOR VALUES (0, 500, 1000)
+GO
+
+CREATE PARTITION FUNCTION [PartitionFunctionHavingDifferentCollationInput](nvarchar(4000)) AS RANGE RIGHT FOR VALUES (N'châu phố', N'Jane Smith', N'John Doe', N'Müller GmbH')
+GO
+
+CREATE PARTITION FUNCTION [PartitionFunctionNameGreaterThan64AndLessThan128abcdefghijklmnopqrstuvwxyz](bigint) AS RANGE RIGHT FOR VALUES (0, 100, 1000, 10000)
+GO
+
+CREATE PARTITION FUNCTION [PartitionFunctionNameGreaterThan64AndLessThan128HavingEmoji😎$123🌍rder](bigint) AS RANGE RIGHT FOR VALUES (0, 100, 1000, 10000)
+GO
+
+CREATE PARTITION FUNCTION [SmallDateTimePartitionFunction](smalldatetime) AS RANGE RIGHT FOR VALUES (N'2019-01-01T00:00:00.000', N'2022-01-01T00:00:00.000', N'2022-07-01T00:00:00.000')
+GO
+
+CREATE PARTITION FUNCTION [SmallIntPartitionFunction](smallint) AS RANGE RIGHT FOR VALUES (-32768, 0, 32767)
+GO
+
+CREATE PARTITION FUNCTION [SmallMoneyPartitionFunction](smallmoney) AS RANGE RIGHT FOR VALUES (2.5679, 3.5679, 4.9100)
+GO
+
+CREATE PARTITION FUNCTION [SqlVariantPartitionFunction](sql_variant) AS RANGE RIGHT FOR VALUES (N'abc  ', N'Apple', N'Banana', N'Some text', N'xyz  ')
+GO
+
+CREATE PARTITION FUNCTION [TinyIntPartitionFunction](tinyint) AS RANGE RIGHT FOR VALUES (0x00, 0x80, 0xFF)
```

Comment:
Question: You mentioned `sys.tinyint` in your definition , while here it reflects `tinyint` . Is this expected?

Reply:
yes, input file was supposed to test type with explicit schema should also work.